tests/cypress/integration/stateWorkflow/stateSubmissionForm/submissionType.spec.ts

```diff
@@ -9,6 +9,7 @@ describe('submission type', () => {
             const pathnameArray = pathname.split('/')
             const draftSubmissionId = pathnameArray[2]
             cy.visit(`/submissions/${draftSubmissionId}/type`)
+            cy.findByRole('heading', { level: 2, name: /Submission type/, timeout: 10000 })
```

Comment:
Why not wait for the query to return here also? The problem with cy.visit to specific form page is it bypasses those wait function which do address flakes. I bet we could have flakes here in other tests as well that use cy.visit to jump ahead in the form